packages/krino/krino/region/Akri_Startup.cpp

```diff
@@ -84,7 +84,7 @@ Startup::Startup(int argc, char ** argv)
     throw std::runtime_error("MPI_Init failed");
   }
 
-  stk::EnvData::instance().m_parallelComm = MPI_COMM_WORLD;
```

Comment:
This function calls `MPI_Init` so it makes sense to use `MPI_COMM_WORLD` here (there's no comm split happening)